src/lib/deploy.js

```diff
@@ -554,10 +564,10 @@ async function findSmartContracts(path) {
  * @param {object} deploy  The build/build.json in object format.
  * @returns {string}       The smart contract name.
  */
-function chooseSmartContract(config, deploy, network) {
+function chooseSmartContract(config, deploy, deployAliasesConfigName, network) {
```

Comment:
The above suggestion would clean this up as well b/c `deployAliasesConfigName` wouldn't need to be passed in; `network` would also become `deployAliasName`.